.idea/.gitignore

```diff
@@ -0,0 +1,2 @@
+# Default ignored files
+/workspace.xml
```

Comment:
This can go in the root .gitignore? any reason for this?

Reply:
Don't know how it got here

Reply:
Ah, the /.idea directory should not be included at all